Resources/public/templates/fields/edit/binaryfile.hbt

```diff
@@ -8,33 +8,47 @@
             <p class="ez-field-description is-showing-description">
                 {{ translate_property fieldDefinition.descriptions }}
             </p>
-            <p class="ez-binarybase-help">Drag and drop your file here or simply click on the buttons and select your file manually.</p>
+            <p class="ez-binarybase-help">
+                {{translate "binaryfile.help" "fieldedit"}}
+            </p>
         </label>
     </div>
     <div class="pure-u ez-editfield-input-area">
         <div class="ez-editfield-input ez-binarybase-drop-area"><div class="ez-binaryfile-input-ui">
             <div class="ez-binaryfile-content ez-font-icon ez-binarybase-content">
                 <ul class="ez-binaryfile-properties">
-                    <li><b>File name:</b> <span class="ez-binaryfile-properties-name">{{ binaryfile.name }}</span></li>
-                    <li><b>Type:</b> <span class="ez-binaryfile-properties-type">{{ binaryfile.type }}</span></li>
-                    <li><b>File size:</b> <span class="ez-binaryfile-properties-size">{{ binaryfile.size }}</span>&nbsp;bytes</li>
-                    <li class="ez-binaryfile-properties-download"><a href="{{ binaryfile.uri }}" class="ez-binaryfile-download" target="_blank">Download the file</a></li>
+                    <li>
+                        <b>{{translate "binaryfile.name" "fieldedit"}}</b>
+                        <span class="ez-binaryfile-properties-name">{{ binaryfile.name }}</span>
+                    </li>
+                    <li>
+                        <b>{{translate "binaryfile.type" "fieldedit"}}</b>
+                        <span class="ez-binaryfile-properties-type">{{ binaryfile.type }}</span>
+                    </li>
+                    <li>
+                        <b>{{translate "binaryfile.size" "fieldedit"}}</b>
+                        <span class="ez-binaryfile-properties-size">{{ binaryfile.size }}</span>&nbsp;
+                        {{translate "binaryfile.bites" "fieldedit"}}
```

Comment:
actually we should probably have a string with a parameter for the size, so until we have that, we should let the original string